packages/horizon/contracts/data-service/interfaces/IDataService.sol

```diff
@@ -0,0 +1,139 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.26;
+
+import { IGraphPayments } from "../../interfaces/IGraphPayments.sol";
+
+/**
+ * @title Interface of the base {DataService} contract as defined by the Graph Horizon specification.
+ * @notice This interface provides a guardrail for contracts that use the Data Service framework
+ * to implement a data service on Graph Horizon. Much of the specification is intentionally loose
+ * to allow for greater flexibility when designing a data service. It's not possible to guarantee that
+ * an implementation will honor the Data Service framework guidelines so it's advised to always review
+ * the implementation code and the documentation.
+ * @dev This interface is expected to be inherited and extended by a data service interface. It can be
+ * used to interact with it however it's advised to use the more specific parent interface.
+ */
+interface IDataService {
+    /**
+     * @notice Emitted when a service provider is registered with the data service.
+     * @param serviceProvider The address of the service provider.
+     */
+    event ServiceProviderRegistered(address indexed serviceProvider);
+
+    /**
+     * @notice Emitted when a service provider accepts a provision in {Graph Horizon staking contract}.
+     * @param serviceProvider The address of the service provider.
+     */
+    event ProvisionAccepted(address indexed serviceProvider);
+
+    /**
+     * @notice Emitted when a service provider starts providing the service.
+     * @param serviceProvider The address of the service provider.
+     */
+    event ServiceStarted(address indexed serviceProvider);
```

Comment:
Same here, providing additional data would allow indexing more information about the service